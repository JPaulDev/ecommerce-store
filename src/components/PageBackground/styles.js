import styled from 'styled-components';

export const Wrapper = styled.div`
  height: 100%;
  position: absolute;
  width: 100%;
  z-index: -100;
`;

export const Background = styled.div`
  background-color: var(--color-grey-9);
  background-image: url(${({ background }) => background});
  background-position: center top;
  background-repeat: no-repeat;
  height: 100vh;
  position: sticky;
  top: 0;
`;
