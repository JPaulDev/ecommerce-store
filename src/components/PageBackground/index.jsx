import * as Styled from './styles';

export default function PageBackground({ background }) {
  return (
    <Styled.Wrapper>
      <Styled.Background background={background} />
    </Styled.Wrapper>
  );
}
