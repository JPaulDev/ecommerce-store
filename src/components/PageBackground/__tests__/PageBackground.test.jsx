import renderer from 'react-test-renderer';
import PageBackground from '../index';

describe('PageBackground component', () => {
  it('renders a page background image', () => {
    const tree = renderer.create(<PageBackground />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
