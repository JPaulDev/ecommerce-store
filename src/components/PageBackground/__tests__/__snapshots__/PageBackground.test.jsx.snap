// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageBackground component renders a page background image 1`] = `
<div
  className="sc-bczRLJ yCwRa"
>
  <div
    className="sc-gsnTZi dAbdwD"
  />
</div>
`;
